internal/workflow.go

```diff
@@ -945,6 +945,8 @@ func (wc *workflowEnvironmentInterceptor) ExecuteChildWorkflow(ctx Context, chil
 // WorkflowInfo information about currently executing workflow
 type WorkflowInfo struct {
 	WorkflowExecution        WorkflowExecution
+	OriginalRunID            string // The original runID before resetting. Using it instead of current runID can make workflow decision determinstic after reset. See also FirstRunId
+	FirstRunID               string // The very first original RunId of the current Workflow Execution preserved along the chain of ContinueAsNew, Retry, Cron and Reset. Identifies the whole Runs chain of Workflow Execution.
```

Comment:
Should document fields above the field

Reply:
Fixed with https://github.com/temporalio/sdk-go/pull/1090/commits/b4d8ee53878cf0b410202c456872a5bab5827164